jira_sync/config/model.py

```diff
@@ -14,6 +14,12 @@ class StatusesConfig(BaseModel):
     blocked: str
     closed: str
 
+    def values(self):
+        """
+        Return list of values.
+        """
+        return [self.new, self.assigned, self.blocked, self.closed]
+
```

Comment:
Pydantic models already provide this info in, e.g. as the keys of `StatusesConfig.model_fields`, so we don’t need to add this method …

Reply:
Didn't know about that, let me rewrite it.

Reply:
Argh, got that one wrong 😬